install_config/configuring_ephemeral.adoc

```diff
@@ -21,26 +21,28 @@ discussed here].
 Ephemeral storage management permits administrators to limit the
 resources consumed by individual pods and containers, and for pods and
 containers to specify requests and limits on their use of such
-ephemeral storage.  It is a tech preview new in {product-title} 3.10
+ephemeral storage. This is a technology preview for {product-title} 3.10
 and is disabled by default.
 
-Note that this tech preview does not change any of the mechanisms for
+[NOTE]
+====
+This technology preview does not change any of the mechanisms for
 making local storage available in {product-title}; the existing
-mechanisms (root directory or runtime directory) are still used.  This
-preview only provides mechanism for managing the use of this resource.
+mechanisms (root directory or runtime directory) are still used. This
+technology preview only provides a mechanism for managing the use of this resource.
+====
 
 [[ephemeral-storage-enabling-ephemeral-storage]]
-=== Enable Ephemeral Storage
-Enable the `LocalStorageCapacityIsolation` feature gate on all masters and nodes.
+== Enabling ephemeral storage
```

Comment:
Please update the case on line 2 and consider making the title on 12 more descriptive.